source/Features/OverrideStatTooltips/Stats/DurabilityStat.cs

```diff
@@ -13,13 +13,13 @@ public void SetupTooltip(StatTooltipData tooltipData, MechDef mechDef)
         {
             var value = mechDef.MechDefAssignedArmor;
             value *= ArmorMultiplier(mechDef);
-            value *= DamageReductionMultiplierAll(mechDef);
+            //value *= DamageReductionMultiplierAll(mechDef);
             tooltipData.dataList.Add("<u>" + Strings.T("Armor") + "</u>", $"{value}");
         }
 
         {
-            var value = DamageReductionMultiplierAll(mechDef);
-            tooltipData.dataList.Add("<u>" + Strings.T("Damage Reduction") + "</u>", Strings.T("{0} %", value));
+            var value = 1 - DamageReductionMultiplierAll(mechDef);
+            tooltipData.dataList.Add("<u>" + Strings.T("Damage Reduction") + "</u>", Strings.T($"{value * 100} %"));
```

Comment:
```suggestion
            tooltipData.dataList.Add("<u>" + Strings.T("Damage Reduction") + "</u>", Strings.T("{0} %", percent));
```
going through string.T formatting should in theory allow internationalization on the string.. in this case the percent sign and the whitepace in between...

Reply:
Doing it this way
```csharp
var reduction = 1 - DamageReductionMultiplierAll(mechDef);
var percent = (int)(reduction * 100);
tooltipData.dataList.Add("<u>" + Strings.T("Damage Reduction") + "</u>", Strings.T("{0} %", percent));
```

Results in floating point errors with some reduction values that will cause incorrect reduction to display (EX: in a 0.8 damage reduction test I had this display as 19% damage reduction). I recommend not doing an integer conversion and using standard percent numeric formatting to handle this instead. 

```csharp
var reduction = 1 - DamageReductionMultiplierAll(mechDef);
tooltipData.dataList.Add("<u>" + Strings.T("Damage Reduction") + "</u>", Strings.T("{0:P0}", reduction));
```

Reply:
Looks much better, implement that, squash everything, and I'll merge

thx